rust/parquet/src/arrow/schema.rs

```diff
@@ -409,13 +413,15 @@ fn arrow_to_parquet_type(field: &Field) -> Result<Type> {
                 .with_length(*length)
                 .build()
         }
-        DataType::Decimal(precision, _) => Type::primitive_type_builder(
-            name,
-            PhysicalType::FIXED_LEN_BYTE_ARRAY,
-        )
-        .with_repetition(repetition)
-        .with_length((10.0_f64.powi(*precision as i32).log2() / 8.0).ceil() as i32)
-        .build(),
+        DataType::Decimal(precision, scale) => {
+            Type::primitive_type_builder(name, PhysicalType::FIXED_LEN_BYTE_ARRAY)
```

Comment:
Are we taking the option of always writing decimals as FLBA? I'm fine with that, just confirming.
I don't know under what conditions one would want to use i32 or i64. Is it related to precision and scale?

Reply:
Fixed Length Byte Array makes sense to me (as the interpretation of the bytes for decimal is different than either i32 or i64). 

Reasons I could imagine using i32 or i64 to write decimals into Parquet would be 
1. ecosystem compatibility (aka that the pandas parquet reader assumed decimals were stored using those types) 
2.possibly so better / more performant encodings could be used.

But I am just SWAG'ing it here